oc_mirror.sh

```diff
@@ -81,3 +78,74 @@ EOF
    popd
 
 }
+
+function set_full_imageset() {
+
+   # Create imageset with all images from releaseImage
+   imageset=$1
+
+   latest_release=$(oc-mirror list releases --channel candidate-${OPENSHIFT_RELEASE_STREAM}| tail -n1)
+
+   cat > "${imageset}" << EOF
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+archiveSize: 16
+storageConfig:
+  local:
+    path: metadata
+mirror:
+  platform:
+    architectures:
+      - "amd64"
```

Comment:
Q: Any specific reason to specify just here the architecture?

Reply:
Added to other imageset